strings/steve.json

```diff
@@ -0,0 +1,20 @@
+{
+    "ballmer_good_msg": [
+        "Hey guys, what if we rolled out a bluespace wiring system so mice can't destroy the powergrid anymore?",
+        "Hear me out here. What if, and this is just a theory, we made R&D controllable from our PDAs?",
+        "I'm thinking we should roll out a git repository for our research under the AGPLv3 license so that we can share it among the other stations freely.",
+        "I dunno about you guys, but IDs and PDAs being separate is clunky as fuck. Maybe we should merge them into a chip in our arms? That way they can't be stolen easily.",
+        "Why the fuck aren't we just making every pair of shoes into galoshes? We have the technology."
+    ],
+
+    "ballmer_windows_me_msg": [
+        "Yo man, what if, we like, uh, put a webserver that's automatically turned on with default admin passwords into every PDA?",
+        "So like, you know how we separate our codebase from the master copy that runs on our consumer boxes? What if we merged the two and undid the separation between codebase and server?",
+        "Dude, radical idea: H.O.N.K mechs but with no bananium required.",
```

Comment:
no period on the K

Reply:
I do not intend to edit these strings at this time, these are copied verbatim with all of the spelling and grammar mistakes

Reply:
i think a correction can be forgiven, it's not gonna be a slippery slope from a dot